apps/stripe/src/app/api/saleor/payment-gateway-initialize-session/use-case-response.test.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, it } from "vitest";
+
+import { stripePublishableKey } from "@/__tests__/mocks/stripe-publishable-key";
+
+import { PaymentGatewayInitializeSessionUseCaseResponses } from "./use-case-response";
+
+describe("PaymentGatewayInitializeSessionUseCaseResponses", () => {
+  describe("Success", () => {
+    it("should return fetch API response with status code and message", async () => {
```

Comment:
```suggestion
    it("getResponse() returns valid Response with status 200 and formatted 'data' object containing Stripe PK", async () => {
```

I think Response is more obvious that "fetch API response" (I am editing this comment because I didnt understand it before)